scipy/signal/windows/_windows.py

```diff
@@ -2115,6 +2210,7 @@ def _fftautocorr(x):
     ('taylor', 'taylorwin'): (taylor, False),
     ('triangle', 'triang', 'tri'): (triang, False),
     ('tukey', 'tuk'): (tukey, False),
+    ('lanczos', 'sinc'): (lanczos, False),
```

Comment:
Move up to keep this alphabetical. I think it might be what is making this no longer alphabetical:

https://output.circle-artifacts.com/output/job/db396e8f-2f10-4b9d-ab9a-4d66b23081ba/artifacts/0/html/reference/signal.windows.html#module-scipy.signal.windows

Reply:
@larsoner Thank you for reviewing.  I fixed it. Ref: https://output.circle-artifacts.com/output/job/e07ff45d-6404-4b9d-8e05-ee2fed3f0227/artifacts/0/html/reference/signal.windows.html#module-scipy.signal.windows